01week/resume/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+  <html>
+  <head>
+    <title>Resume</title>
+  </head>
+  <link type="text/css" rel="stylesheet" href="stylesheet.css"/>
+  <body style="margin-top:80px; margin-left:80px; margin-right:80px;
+  margin-bottom:80px;">
```

Comment:
If you link this html file to a css file, you can do away with the all these properties and values (i.e. - margin-top:80px).